python-frontend/typeshed_serializer/tests/test_tox_runner.py

```diff
@@ -0,0 +1,219 @@
+#
```

Comment:
It looks like coverage has not been taken into account by SonarQube analysis.
I think that's because the configuration in the `python-frontend/pom.xml` is the following:
```xml
    <sonar.sources>pom.xml,src/main/java,typeshed_serializer/serializer</sonar.sources>
    <sonar.python.coverage.reportPaths>typeshed_serializer/cov.xml</sonar.python.coverage.reportPaths>
    <sonar.python.version>3.9</sonar.python.version>
```

So I think we should:
- either update `sonar.sources` to include `typeshed_serializer/runners`
- or move `runners` inside `typeshed_serializer/serializer`